### PR TITLE
events: performance improvements for big data

### PR DESCRIPTION
**Pull request by smatvienko-tb**
UI fast response with latest events, TTL cleanup speedup (with schema update)
Comments provided inside schema update file.
Update schema implemented from release 3.2.2

### CODE REVIEW

File: application/src/main/data/upgrade/3.2.2/schema_update_event.sql [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+COMMENT ON INDEX public.idx_event_ts
+    IS 'This index helps to delete events by TTL using timestamp';
+
+
+-- Index: idx_event_tenant_entity_type_entity_event_type_created_time_des
+
+DROP INDEX IF EXISTS public.idx_event_tenant_entity_type_entity_event_type_created_time_des;
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_event_tenant_entity_type_entity_event_type_created_time_des
```
Comment (ashvayka) [inline application/src/main/data/upgrade/3.2.2/schema_update_event.sql:79]: missing ;

Comment (ashvayka) [reply application/src/main/data/upgrade/3.2.2/schema_update_event.sql:79]: and index body

File: application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java [hunk 2]
```diff
@@ -460,6 +460,9 @@ public void upgradeDatabase(String fromVersion) throws Exception {
                     loadSql(schemaUpdateFile, conn);
                     log.info("Edge TTL functions successfully loaded!");
                     conn.createStatement().execute("UPDATE tb_schema_settings SET schema_version = 3003000;");
+                    log.info("Updating indexes and TTL procedure for event table...");
```
Comment (ashvayka) [inline application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java:463]: Execute before set schema version.

File: application/src/main/data/upgrade/3.2.2/schema_update_event.sql [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+$BODY$;
+
+
+-- Index: idx_event_ts
+
+DROP INDEX IF EXISTS public.idx_event_ts;
+
+-- Hint: add CONCURRENTLY to CREATE INDEX query in case of more then 1 million records or during live update
+-- CREATE INDEX CONCURRENTLY IF NOT EXISTS idx_event_ts
+CREATE INDEX IF NOT EXISTS idx_event_ts
```
Comment (ashvayka) [inline application/src/main/data/upgrade/3.2.2/schema_update_event.sql:66]: add to schema-entities-idx.sql

File: application/src/main/data/upgrade/3.2.2/schema_update_event.sql (hunk 1 above)
Comment (ashvayka) [reply application/src/main/data/upgrade/3.2.2/schema_update_event.sql:79]: add to schema-entities-idx.sql

Comment (smatvienko-tb) [reply application/src/main/data/upgrade/3.2.2/schema_update_event.sql:79]: corrected. it is an example of how to build an index concurrently. Marked as a comment.

File: application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java (hunk 2 above)
Comment (smatvienko-tb) [reply application/src/main/java/org/thingsboard/server/service/install/SqlDatabaseUpgradeService.java:463]: done

File: application/src/main/data/upgrade/3.2.2/schema_update_event.sql (hunk 3 above)
Comment (smatvienko-tb) [reply application/src/main/data/upgrade/3.2.2/schema_update_event.sql:66]: done

**Comment by smatvienko-tb** [comment]
@andrew the hsqldb for tests does not support postgres-specific features for index creating
image

**Pull request merged**